Thegaijin/reacipe
# #155024525 Add create category feature

What does this PR do?
Add user login functionality

#### Description of the task to be completed
The user should be able to create a category in the app

#### How should this be manually tested?
Assuming the user is already logged in, on the dashboard they will be able to see the an `add category` button. When you click on that button, a modal will pop up and the user can enter the category name and description. After that click on submit then close the modal. The category has now been created.

What are the relevant pivotal tracker stories?
[#155024525](https://www.pivotaltracker.com/story/show/155024525)
